solidity-landing/layouts/default.vue

```diff
@@ -1,9 +1,8 @@
-<script lang="ts" setup></script>
```

Comment:
the name of the `vue` component must be in the camel case. `DefaultLayout`

Reply:
reserved nuxt name